src/pages/historypage/historycard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {AiOutlineHeart,AiOutlineClose} from "react-icons/ai"
+import {MdOutlinePlaylistPlay,MdOutlineWatchLater} from "react-icons/md"
+import {useHistory} from "./../../context/index"
+const HistoryCard = ({product,index}) => {
+    const{historyDispatch} = useHistory()
+    const{_id,title,description,charactor} = product
+
+  return (
+<div class="border-skin text-overlay-card-dimension card-relative video-card" key = {index}>
+    <div class="text-overlay-card-img-box">
+      <img
+      src={`https://i.ytimg.com/vi/${_id}/hq720.jpg`} class="history_media"
+    />
+    </div>
+    {/* <div class="card-head head-overflow">{title}</div> */}
+    <AiOutlineClose color="#AB542F" size="4rem" class="card_close"
```

Comment:
I prefer sizing icons using 'em' units, cause they are relative to parent's font size.